### ISSUE TITLE
Latest version doesn't 'see' the Classic S200

### ISSUE THREAD

**Issue by hansw1204** [pschroeder89/homebridge-levoit-humidifiers#64]
**Describe The Bug:**
With the latest version, the humidifier disappears from HomeBridge and HK and in the log there is a reference that the device cannot be found on the WiFi. At the same time it can be controlled by the VeSynch app. Re-installing the previous version brings the device back to HB and HK.

**To Reproduce:**
Install the latest plugin version

**Expected behavior:**

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by pschroeder89** [comment]
This should be fixed in 1.12.1, sorry about that.

**Comment by hansw1204** [comment]
Thank you Paul!

Things happen, I understand. I appreciate your work!

Best wishes,

Hans


From: Paul Schroeder ***@***.***>
Date: Tuesday, January 16, 2024 at 10:25
To: pschroeder89/homebridge-levoit-humidifiers ***@***.***>
Cc: HaWi ***@***.***>, Author ***@***.***>
Subject: Re: [pschroeder89/homebridge-levoit-humidifiers] Latest version doesn't 'see' the Classic S200 (Issue #64)

This should be fixed in 1.12.1, sorry about that.
